src/popup/components/Modals/ConfirmTransactionSign.vue

```diff
@@ -321,7 +346,42 @@ export default defineComponent({
       popupProps.value?.reject(new RejectedByUserError());
     }
 
-    onMounted(async () => {
+    async function verifyTransaction() {
+      if (popupProps.value?.txBase64 && popupProps.value.tx) {
+        try {
+          verifying.value = true;
+          const { balance, updateBalances } = useBalances({ store });
+
+          await updateBalances();
```

Comment:
Updating the balance of only the active account is required here not all accounts.

Reply:
Shall we maybe consider it a future improvement, since other places where we are using `updateBalances` except `balances.ts` also requiring updating of only active account balance?

Reply:
Asking for a `balances` from the `balances.ts` is already applying `updateBalances` every 3 seconds. 
- Maybe you think that current concerns are worth not to use `balances` composable at all here, and just to fetch the active account balance directly, even without sdk initialization?
- Or you are suggesting to add additional flag to the `balances` composable to fetch the balance only for an active account. But will the current logic in `initPollingWatcher` be able to load run `updateBalances` in one component and `updateActiveAccountBalance` in another component in the same scope (parent component).

Reply:
> Maybe you think that current concerns are worth not to use balances composable at all here, and just to fetch the active account balance directly, even without sdk initialization?

This sounds more reasonable.
Again, not expecting over-engineering in the name of optimization but I guess we can achieve this without using `balances` composable with periodic watchers. Just saw the code, `balance` composable is being called `onMounted` hook. Hence it may not be adding significant delay.

Reply:
I think this makes sense to use `aeSdk.getBalance` here instead of fetching balances for all the accounts.

Reply:
Updating the balance of only the active account is required in every place we are using `updateBalances` currently. So what would the general suggestion be?